contract/src/main/java/com/ritense/valtimo/contract/json/patch/JsonPatchBuilder.java

```diff
@@ -93,28 +93,45 @@ public JsonPatchBuilder addJsonNodeValue(JsonNode destination, JsonPointer path,
     }
 
     /**
-     * This will adjust the first unindexed array position ('/-') to a new index depending on previous operations or destination object.
+     * This will adjust the first un-indexed array position ('/-' or '/+') to an index depending on previous operations or destination object.
+     * The '/-' will add the JSON to a new element in the array.
+     * The '/+' will append JSON to the last element in the array, or create a new element.
```

Comment:
This line could be improved to say in which case a new element is created.

Reply:
Change comment